index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>CV</title>
+    <link rel="icon" href="./img/favicon.png">
+</head>
+<body>
+<header>
+    <h1>Kiryl Pliushchenia</h1>
+    <figure>
+        <img src="./img/avatar.jpg" alt="my avatar" width="300em">
+        <figcaption>Kiryl Pliushchenia</figcaption>
+    </figure>
+    <nav>
+    <a href="#"> VK</a>
+    <a href="#"> Instagram</a>
+    <a href="#"> Facebook</a>
+</nav>
```

Comment:
Please, use code styling and add some reasonable spaces to have a logical structure.
Remove empty space inside a-tags. If you need some space, please use styles.